index.js

```diff
@@ -1,9 +1,35 @@
 'use strict'
+const path = require('path')
```

Comment:
lol i hope we can have a semicolons discussion at some point (not in this PR)

Reply:
@ljharb not sure what you mean by the semicolons discussion but the current linter config prohibits their use in most places

Reply:
Right, I’m saying I’d like to discuss changing that (very nonstandard compared to the ecosystem) linter config, but i don’t want to derail PRs with it :-)

Reply:
ok, makes sense.

Reply:
Can we have this conversation in slack?  I also want semi colons, but the work it takes to maintain a standard fork is non-trivial.  I had a short chat with Feross on twitter about this, and I think we could automate a fork. I just haven't had time to dig in.

Reply:
Yes, let's chat about it in slack. (The solution here is to use `eslint --fix`, so we match the majority of the ecosystem and also aren't tied to unidiomatic configuration choices)

Reply:
We chatted in slack. The conclusion is that I will do whatever the rest of the group agrees. Closing this for now.